### PR TITLE
Initial web pages for Data Release 10

### PR DESCRIPTION
**Pull request by geordie666**
This PR adds the new web pages for DR10. The resulting pages are still missing a few details: 

- A description of the syntax for the cutout server from @dstndstn.
- A description of the sky pattern correction from @rongpu.

But they're probably "good enough" for an initial release.

### CODE REVIEW

**Comment by geordie666** [comment]
This PR will address #160.

File: CHANGES.md [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
-- *Planned*: correct units for `ccdskycounts`
+## 10.0.0 (DR10, unreleased)
+
+- Near-final updates for DR10
+  ([PR#165](https://github.com/legacysurvey/legacysurvey/pull/165)):
+    - Includes a sufficient set of DR10 pages for an initial release.
+    - Still missing details of the syntax for the cutout server.
+    - Still missing details about the DR10 sky pattern correction.
+    - Addresses [issue #160](https://github.com/legacysurvey/legacysurvey/issues/160).
+- *Planned for DR9*: correct units for `ccdskycounts`
```
Comment (weaverba137) [inline CHANGES.md:11]: Does this correspond to the changes in units `erg...` -> `erg/s...` that I saw in many of the files?  If so we should remove the "Planned for DR9".  If not, then this should not be included  in the changes for DR10.

File: CHANGES.md [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
   ([#152](https://github.com/legacysurvey/legacysurvey/issues/152)).
-- *Planned*: actually update the raw and CP-processed data access instructions
+- *Planned for DR9*: actually update the raw and CP-processed data access instructions
   ([#143](https://github.com/legacysurvey/legacysurvey/issues/143)).
-- *Planned*: DR9 gallery ([#117](https://github.com/legacysurvey/legacysurvey/issues/117)).
+- *Planned for DR9*: DR9 gallery ([#117](https://github.com/legacysurvey/legacysurvey/issues/117)).
```
Comment (weaverba137) [inline CHANGES.md:15]: I think "Planned for DR9" is no longer appropriate.  If it has not actually been implemented, it should not be included in the changes for DR10.

**Review by weaverba137 — CHANGES_REQUESTED** [review]
I've found a few items so far, mostly having to do with meta-issues rather than the actual content.

File: pages/dr10/files.rst [hunk 3]
```diff
@@ -0,0 +1,1066 @@
...
+.. contents::
+
+Directory Structures
+====================
+
+For Web Access
+--------------
+
+| **Top level directory:**
+| https://portal.nersc.gov/cfs/cosmo/data/legacysurvey/dr10/south/
```
Comment (weaverba137) [inline pages/dr10/files.rst:27]: This is not actually the top level directory.

File: pages/dr10/files.rst [hunk 4]
```diff
@@ -0,0 +1,1066 @@
...
+| **Top level directory for sweep catalogs:**
+| https://portal.nersc.gov/cfs/cosmo/data/legacysurvey/dr10/south/sweep/
+
+
+At NERSC (for collaborators)
+----------------------------
+
+| **Top level directory:**
+| /global/cfs/cdirs/cosmo/data/legacysurvey/dr10/south/
```
Comment (weaverba137) [inline pages/dr10/files.rst:36]: Again, this is not the actual top-level directory.

File: pages/contact.rst [hunk 5]
```diff
@@ -44,10 +44,19 @@ NOIRLab Community Pipeline:
 WISE Reductions:
     * Aaron Meisner  aaron.meisner@noirlab.edu
 
-Run Managers
+DR10 Run Managers
+    * Jan Snigula (snigula@mpe.mpg.de)
```
[the remark this answers is not in the document]
Comment (geordie666) [reply pages/contact.rst]: I can't think of a reason! I switched to using angle brackets around all email addresses on the contacts page.

File: CHANGES.md (hunk 1 above)
Comment (geordie666) [reply CHANGES.md]: No, this is a different issue. In any case, I removed all of the "Planned for DR9" comments.

File: CHANGES.md [hunk 6]
```diff
@@ -1,12 +1,18 @@
...
+- Near-final updates for DR10
+  ([PR#165](https://github.com/legacysurvey/legacysurvey/pull/165)):
+    - Includes a sufficient set of DR10 pages for an initial release.
+    - Still missing details of the syntax for the cutout server.
+    - Still missing details about the DR10 sky pattern correction.
+    - Addresses [issue #160](https://github.com/legacysurvey/legacysurvey/issues/160).
+- *Planned for DR9*: correct units for `ccdskycounts`
   ([#152](https://github.com/legacysurvey/legacysurvey/issues/152)).
-- *Planned*: actually update the raw and CP-processed data access instructions
+- *Planned for DR9*: actually update the raw and CP-processed data access instructions
```
[the remark this answers is not in the document]
Comment (geordie666) [reply CHANGES.md]: I removed all of the "Planned for DR9" comments.

File: CHANGES.md (hunk 2 above)
Comment (geordie666) [reply CHANGES.md]: I removed all of the "Planned for DR9" comments.

File: pages/dr10/files.rst (hunk 3 above)
Comment (geordie666) [reply pages/dr10/files.rst]: Thanks — good spot. I changed this to `https://portal.nersc.gov/cfs/cosmo/data/legacysurvey/dr10/`.

File: pages/dr10/files.rst (hunk 4 above)
Comment (geordie666) [reply pages/dr10/files.rst]: Again, thanks for noticing this oversight. I changed this to `/global/cfs/cdirs/cosmo/data/legacysurvey/dr10/`.

**Review by weaverba137 — APPROVED** [review]

**Comment by weaverba137** [comment]
I made some comments on some of the other open issues, which we should at least figure out whether those have been addressed or not. I don't see anything further in the PR itself though to hold it up.  I'm working a few timezones ahead, so it would be reasonable to give other collaborators a chance to look.

File: CHANGES.md [hunk 7]
```diff
@@ -1,12 +1,13 @@
 # legacysurvey Change Log
 
-## 9.2.0 (DR9, unreleased)
-
-- *Planned*: correct units for `ccdskycounts`
```
Comment (weaverba137) [inline CHANGES.md:5]: We should leave these notes as "planned" until they are actually implemented, rather than deleting them entirely.

Comment (geordie666) [reply CHANGES.md:5]: OK, I added:

Planned: correct units for ccdskycounts (#152).
Planned: actually update the raw and CP-processed data access instructions (#143).

back into the changes docs.

Comment (geordie666) [reply CHANGES.md:5]: I _didn't_ retain the point about a gallery being planned for DR9, as I think we've moved past that issue.

Comment (weaverba137) [reply CHANGES.md:5]: Yes, that's fair.

**Comment by dstndstn** [comment]
Hi, I just reinstated the section about the viewer cutout service; I believe it's all working as described now.

**Comment by weaverba137** [comment]
@dstndstn, thank you, but unfortunately the tests are now failing.  I'll see if I can fix it, but if you happen to know exactly what you changed, please go ahead.

**Comment by dstndstn** [comment]
sorrry!, will try to fix asap.

**Comment by weaverba137** [comment]
It should be pretty easy: I think you inadvertently deleted a link that still exists in one or more of dr1/description.rst or dr2/description.rst.

**Comment by weaverba137** [comment]
svtips might also have a broken link.

**Comment by weaverba137** [comment]
I fixed the minor issue with `svtips.rst` already, thank you @dstndstn.

**Comment by geordie666** [comment]
After the updates from @dstndstn I found a few more broken links. These pointed to `dr10-early` in the viewer, which no longer exists since Dustin upgraded to `dr10` proper.

Now that I've fixed these cases, I'm ready to merge. I'll do so (very) early on Tuesday if nobody beats me to it — in particular, if @weaverba137 wants to merge this PR sooner to serve a version of the webpages that's fine by me.

**Comment by weaverba137** [comment]
It would be tricky for me to tag and deploy on Tuesday, I am on vacation after all, but if you want to see this up on Monday or Wednesday that would be a little bit easier.  Also, it wouldn't hurt to email the DECaLS list to see when the rest of the team is expecting this to go live.

**Pull request merged**